src/index.js

```diff
@@ -1,3 +1,4 @@
-module.exports = {
-  Client: require('./Client')
-};
+import Client from './Client';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Can you disable this rule globally in `.eslintrc`, as named exports will be used quite often?